scanner/src/lib/dataproducts/battery/ManufacturingDataSheet_v0.1.svelte

```diff
@@ -0,0 +1,153 @@
+    }
+    roundTripEfficiency: {
+      initialEnergyEfficieny: number
+      degradedEnergyEfficiency: number
+    }
+    voltageLevels: {
+      nominalVoltage: number
+      maximumVoltage: number
+      minimumVoltage: number
+    }
+    temperatureRange: {
+      minimumTemperature: number
+      maximumTemperature: number
+    }
+    expectedLifetime: {
+      cycleLife: number
+      cycleRate: string
+      referenceTest: string
+    }
+    legalConformity: {
+      requirementConformity: string[]
+      conformityDeclaration: string
+    }
+  }
+
+  function camelCaseToWords(s: string) {
+    const result = s.replace(/([A-Z])/g, " $1")
+    return result.charAt(0).toUpperCase() + result.slice(1)
+  }
+</script>
+
+<article>
+  <DataRow label="Product Name" value={data.productName} />
+  <DataRow label="Battery Model" value={data.batteryModel} />
+  <DataRow label="Battery Category" value={data.batteryCategory} />
+  <DataRow label="Manufacturing Date" value={data.manufacturingDate} />
+  <DataRow label="Weight" value={formatNumber(data.weight, "kg")} />
+  <DataRow label="Capacity" value={formatNumber(data.capacity, "Ah")} />
+  <DataRow label="Power" value={formatNumber(data.power, "Wh")} />
+  <DataRow label="Extinguishing Agents" value={data.extinguishingAgents.join(", ")} />
+  <DataRow label="Warranty" value={data.warranty} />
+  <div class="divider" />
+  <div class="title no-bottom-margin">Manufacturer Location</div>
+  <div class="subtitle">The details of the battery manufacturer</div>
+  {#each Object.entries(data.manufacturerInformation) as [key, value]}
+    <DataRow label={camelCaseToWords(key)} {value} />
+  {/each}
+  <div class="divider" />
+  <div class="title no-bottom-margin">Manufacturing Location</div>
+  <div class="subtitle">The details of the location of the battery manufacturing plant</div>
+  {#each Object.entries(data.manufacturingLocation) as [key, value]}
+    <DataRow label={camelCaseToWords(key)} {value} />
+  {/each}
+  <div class="divider" />
+  <div class="title no-bottom-margin">Round Trip Efficiency</div>
+  <div class="subtitle">The details of the round trip energy efficiency in energy storages</div>
+  {#each Object.entries(data.roundTripEfficiency) as [key, value]}
+    <DataRow label={camelCaseToWords(key)} value={formatNumber(value, "%")} />
+  {/each}
+  <div class="divider" />
+  <div class="title no-bottom-margin">Voltage Levels</div>
+  <div class="subtitle">The details of the voltage levels of the battery</div>
+  {#each Object.entries(data.voltageLevels) as [key, value]}
+    <DataRow label={camelCaseToWords(key)} value={formatNumber(value, "V")} />
+  {/each}
+  <div class="divider" />
+  <div class="title no-bottom-margin">Temperature Range</div>
+  <div class="subtitle">The details of the acceptable temperature values of the battery</div>
+  {#each Object.entries(data.temperatureRange) as [key, value]}
+    <DataRow label={camelCaseToWords(key)} value={formatNumber(value, "C°")} />
```

Comment:
Asked these to be corrected to `°C` in design as well, this is very obviously wrong